chapters/functions.tex

```diff
@@ -2433,6 +2434,7 @@ \subsection{Annotations for External Libraries and Include Files}\label{annotati
     input Real x;
     output Real y;
   external "C"
+    y = ExternalFunc3_ext(x)
```

Comment:
The `external-function-call` is optional in the grammar, so leaving it out could have been intentional.

Reply:
I agree that it is likely intentional that it is missing here. The first function has the declaration to show that it is possible.